CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # App Center SDK for Unity Change Log
 
+## Release 1.4.0
+
+* Updated native SDK versions.
```

Comment:
Please add some details. Ideally, a list of changes in native SDKs, but the link to release notes of SDKs is OK for me. Also, consider specifying what version of native SDK is used now